wallet-server-ui/.gitignore

```diff
@@ -40,6 +40,7 @@ npm-debug.log
 yarn-error.log
```

Comment:
Can this file come off of the PR?

Reply:
I was thinking it might be useful to add the `.prettierrc` line